.gitignore

```diff
@@ -1,3 +1,3 @@
-
+.vscode/
```

Comment:
It feels like this would be something for a global `.gitignore` file. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Sure, no problem